test-VM.nix

```
@@ -12,14 +15,12 @@ let
     # conf = callPackage ./pkgs/config/config.nix { };
     aarch64-none-elf = callPackage ./pkgs/toolchains/aarch64-none-elf-11-3.nix {};
     demos = callPackage ./pkgs/demos/demos.nix {};
-    baremetal = callPackage ./pkgs/guest/baremetal-tf.nix {toolchain = aarch64-none-elf; };
+    baremetal = callPackage ./pkgs/guest/baremetal-tf.nix {toolchain = aarch64-none-elf; plat = platform; list_tests = tests; list_suites = suites; };
```

Having the names standardized, now you can leverage the `inherit` nix attribute. (Using `inherit var;` it's the same as `var = var;`)
```suggestion
    baremetal = callPackage ./pkgs/guest/baremetal-tf.nix {toolchain = aarch64-none-elf; inherit platform; inherit list_tests; inherit list_suites; };
```